config_test.go

```diff
@@ -43,6 +43,18 @@ func TestConfig_ForProperty_ReturnsExpectedConfig(t *testing.T) {
 				ValueOverrides: map[string]any{"bar": "baz"},
 			},
 		},
+		"subproperty is returned with minimal=true if set on parent": {
+			input: &Config{
+				Minimal:        true,
+				ValueOverrides: map[string]any{"foo": map[string]any{"bar": "baz"}},
```

Comment:
I'd remove the ValueOverrides here, as they're not relevant for the test afaik

Reply:
Removed